app/javascript/widget/i18n/locale/en.json

```diff
@@ -22,7 +22,6 @@
     "IN_A_DAY": "Typically replies in a day"
   },
   "START_CONVERSATION": "Start Conversation",
-  "END_CONVERSATION": "End Conversation",
```

Comment:
@muhsin-k Can you revert back this line? 

**Screenshot**

<img width="962" alt="image" src="https://user-images.githubusercontent.com/64252451/163091430-15d9241b-4d32-4d79-a187-9fe604675297.png">